src/styles.css

```diff
@@ -95,6 +95,12 @@ header {
   border: red 1px solid;
  }
 
+ .recipe-image {
```

Comment:
Nice, apply a style to each of the images in the recipe, good to make sure each image is standardized.